cmd/rook/ceph/operator.go

```diff
@@ -79,6 +84,11 @@ func startOperator(cmd *cobra.Command, args []string) error {
 	}
 
 	rookImage := rook.GetOperatorImage(context.Clientset, containerName)
+	rookBaseImageCephVersion, err := rook.GetOperatorBaseImageCephVersion(context)
+	if err != nil {
+		logger.Errorf("failed to get operator base image ceph version. %v", err)
+	}
+	os.Setenv(operatorCephBaseImageVersion, rookBaseImageCephVersion)
```

Comment:
Instead of an env var, how about just setting a package-level var? It makes sense to only load it once since it will never change, but not sure why we need an environment variable if it's not a parameter being passed to the pod spec.

Reply:
By package-level, do you mean calling the `ceph --version` from the package we need it?
Not sure to understand.

If it's an env var we can get it from any package, that's the intent.

Reply:
By a package var I am referring to a `var` declaration that is typically at the top of the source file and can be referenced from any other package. For example, like [these package vars](https://github.com/rook/rook/blob/3ac06a533682eb5e40f954cf7255a9d28fb98ed7/pkg/operator/ceph/operator.go#L58-L67) for enabling the flex driver or discovery daemon. Since there is exactly one of the flex driver or discovery daemon per operator, a single global var can keep track of that setting. Package vars are less preferred than setting a value on some struct, but in rare circumstances like this it can make sense.